apps/grafana-operator/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: prometheus
```

Comment:
It looks as if this chart copies the YAML fully from the original Helm chart. It should should be possible to reference it if it's in a public help repo. We don't want to maintain detailed files if we can avoid it.

Reply:
Maybe its my kind cluster setup, argocd did not install the kustomize yaml, so switched over to copying it .